Log the timestamp (date+time) when logging starts

xsteve opened the pull request:
Hi, I read about your cool module yesterday on python-weekly.

Here is a patch to show the timestamp when logging starts

xsteve commented:
I would really love to see my patch merged

skorokithakis commented on q.py:
```diff
@@ -185,11 +185,17 @@ def add(self, items, sep='', wrap=True):
     def __init__(self):
-        self.writer = self.Writer(self.FileWriter(self.OUTPUT_PATH), self.time)
+        time = self.time
+        self.writer = self.Writer(self.FileWriter(self.OUTPUT_PATH), time)
```
Hmm, what is this change for?

xsteve replied:
It just caches self.time in time. time is used several times in this method.

skorokithakis replied:
Oh, just for ease of typing, you mean? Makes sense.

skorokithakis commented:
I have added a comment, but it looks like a useful feature, so we can merge if @zestyping doesn't have any objections.

xsteve commented:
I added my patch a long time ago.
Today I found out that there was some activity in this repository. This is why I rebased my pull request to the current master.
I use the feature and it is really useful since it shows an entry when I restart my debugged program
